Jenkinsfile

```diff
@@ -14,7 +12,7 @@ node {
     slave {
         withOpenshift {
             withMaven(
-                mavenImage: "maven:${mavenVersion}",
+                mavenImage: "openjdk:8",
```

Comment:
I guess, if we are going to move down that way we need to rename 'mavenImage' to just 'image'